cumulusci/cli/tests/test_service.py

```diff
@@ -156,6 +158,34 @@ def test_service_connect__alias_already_exists(confirm):
     confirm.assert_called_once()
 
 
+@mock.patch("click.confirm")
+def test_service_connect__set_new_service_as_default(confirm):
+    confirm.side_effect = "y"
```

Comment:
Should we test the "n" path as well?

Reply:
This prompt actually led me to realize that I was mocking incorrectly, such that the return values differed from what click.confirm actually returns (a boolean). Thanks Paul, all fixed now!